feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/User.java

```diff
@@ -20,15 +25,19 @@ public User(String username, String password) {
         this.moodHistory = new ArrayList<>();
     }
 
-    public String getUsername() {
-        return username;
-    }
+    public String getUsername() { return username; }
```

Comment:
This is fine, but I think IntelliJ/Android studio can automatically condense getters/setters automatically into one line, so we don't necessarily need to edit the function so it's in one line